Neos.Neos/Classes/Domain/Service/FusionService.php

```diff
@@ -136,57 +137,40 @@ public function createRuntime(TraversableNodeInterface $currentSiteNode, Control
     public function getMergedFusionObjectTree(TraversableNodeInterface $startNode)
     {
         $siteResourcesPackageKey = $this->getSiteForSiteNode($startNode)->getSiteResourcesPackageKey();
-
         $siteRootFusionPathAndFilename = sprintf($this->siteRootFusionPattern, $siteResourcesPackageKey);
-        $siteRootFusionCode = $this->readExternalFusionFile($siteRootFusionPathAndFilename);
-
-        $mergedFusionCode = '';
-        $mergedFusionCode .= $this->generateNodeTypeDefinitions();
-        $mergedFusionCode .= $this->getFusionIncludes($this->prepareAutoIncludeFusion());
-        $mergedFusionCode .= $this->getFusionIncludes($this->prependFusionIncludes);
-        $mergedFusionCode .= $siteRootFusionCode;
-        $mergedFusionCode .= $this->getFusionIncludes($this->appendFusionIncludes);
 
-        return $this->fusionParser->parse($mergedFusionCode, $siteRootFusionPathAndFilename);
```

Comment:
removing here the use of a possible non existing file $siteRootFusionPathAndFilename
fixes
https://github.com/neos/neos-development-collection/pull/3834